test/end-to-end/ant/base/build.xml

```diff
@@ -5,7 +5,9 @@
 
 	<!-- Absolute path of directory where *.xspec files are located -->
 	<property location="${e2e-base.basedir}/../../cases/" name="xspecfiles.dir" />
-	<makeurl file="${xspecfiles.dir}" property="xspecfiles.dir.url" />
```

Comment:
This line was intentionally removed, right? I just wanted to check because it looks unrelated to the rest of the pull request but is next to the content that was added.

Reply:
It's bad that I can't recall what happened here. 🙄 
So I inspected the code. The removed line is verbose and not necessary. Most likely I noticed its verbosity while working on its adjacent lines and removed the line in question silently.  I should have left a commit message regarding the removal.

Reply:
>The removed line is verbose and not necessary.
@AirQuick : Great, thanks for confirming that the removal is desirable and safe.